introduction.md

```diff
@@ -1,2 +1,4 @@
 Hi. Well I don't want to say too much. I am that shy.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?